templates/tools/dockerfile/php_fpm_nginx.include

```diff
@@ -0,0 +1,17 @@
+# Install nginx
+RUN apt-get install -y nginx-extras --no-install-recommends
+
+# Set config
```

Comment:
I'd appreciate a better commend as do not undestand well what this is setting up.

Reply:
Done.